### DISCUSSION TITLE
Get onboarding PR with no internet connection

### DISCUSSION THREAD

**Discussion by sergVedmak** [renovatebot/renovate#11747] [Request Help] [answered]
Hi! I try to use Renovate and get faced a problem:

I run renovate docker image on VM with no internet connection (for security reasons). Renovate try to make onboarding PR, but get new error

<img width="1146" alt="Снимок экрана 2021-09-14 в 19 08 19" src="https://user-images.githubusercontent.com/16452119/133294634-6f2aac87-e4c8-453d-bac7-2b612821add7.png">

There is no connection to registry.npmjs.org and onboarding PR was not created. We have mirror npm registry and I know that I can change `npmrc` option in `renovate.json`, but I can do it only after onboarding PR created.

Is there any way to change url to npm registry before the learning onboarding is started?

**Comment by viceice** [comment, marked as answer]
you can add it to https://docs.renovatebot.com/self-hosted-configuration/#onboardingconfig ?

**Reply by sergVedmak** [reply]
Thank you! That's what I need
